api/tests/unit/domain/services/authentication/pole-emploi-authentication-service_test.js

```diff
@@ -171,6 +171,29 @@ describe('Unit | Domain | Services | pole-emploi-authentication-service', functi
     });
   });
 
+  describe('#getLogoutUrl', function () {
+    it('should return logout url', function () {
+      // given
+      const redirectUri = 'https://example.org/please-redirect-to-pix';
+      const idToken =
+        'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c';
+
+      // when
+      const redirectTarget = poleEmploiAuthenticationService.getLogoutUrl({ idToken, redirectUri });
+
+      // then
+      const parsedRedirectTarget = new URL(redirectTarget);
+      expect(parsedRedirectTarget.protocol).to.equal('http:');
+      expect(parsedRedirectTarget.hostname).to.equal('logout-url.fr');
+
+      const queryParams = parsedRedirectTarget.searchParams;
+      expect(queryParams.get('id_token_hint')).to.equal(
```

Comment:
suggestion : pourquoi ne pas tester juste le résultat de la fonction vu que c'est ce qui est utilisé par le front finalement ?
`   expect(redirectTarget).to.equal(
        'http://logout-url.fr/?id_token_hint=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c&redirect_uri=https%3A%2F%2Fexample.org%2Fplease-redirect-to-pix'
      );`

Reply:
j'ai l'impression qu'on test `new URL` qui est une librairie externe et qu'on essaye en général de ne pas tester